### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GeorgeWL**
** Draft**

Add link to github release, in your fork.

Name it with PR number - PR title:

`415-FixProfile`, and link it so it can be tested

### CODE REVIEW

**Review by pkramme — APPROVED** [review]
Sad, but approved...

File: README.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ Join the beta here: https://play.google.com/apps/testing/com.alorma.github
 
 Gitskarios is an Android Application that allows you to access GitHub, see your profile, search people, star repositories, and more!
 
+## NOTE ##
+
+The Repository Primary Owner, [@alorma](https://github.com/alorma) is soon to leave the project.
```
Comment (alorma) [inline README.md]: Soon?

I leaved the project, please change it to a more definitively :)

**Review by alorma — COMMENTED** [review]
Great, just correct the sentence

**Review by alorma — APPROVED** [review]
Ok

**Pull request merged**